src/app_service/service/node_configuration/service.nim

```diff
@@ -151,10 +151,6 @@ method setBloomLevel*(self: Service, bloomLevel: string): bool =
   return false
 
 method setFleet*(self: Service, fleet: string): bool =
-  if(not self.settingsService.saveFleet(fleet)):
```

Comment:
@richard-ramos this line saves fleet to the local db via `settingService` and keeps struct in setting service updated. From the settings service we have methods `getFleet()` and `getFleetAsString()` which are used in multiple places in the app. So are the values read on those places, after calling `setFleet` from this service, going to be accurate?